chutney/ui/src/app/modules/campaign/components/create-campaign/campaign-edition.component.html

```diff
@@ -93,109 +93,108 @@ <h4 id="myCampaignLabel">
+          </ng-multiselect-dropdown>
+          }
+        </div>
+        }
+      </div>
+    </div>
+    <div class="row">
       <!-- Left side -->
       <div class="col-md-6">
-        <div class="row campaign-title mt-2">
-          <h4>{{ 'campaigns.edition.scenarios.title' | translate }}</h4>
-        </div>
-        <div class="table-responsive">
           <table class="table table-striped table-hover">
+            <thead>
+              <th colspan="4">
+                {{ 'campaigns.edition.scenarios.title' | translate }}
+              </th>
+              <th class="text-center pe-5">Dataset</th>
+              <th></th>
+            </thead>
             <tbody dragula="DRAGGABLE" [(dragulaModel)]="scenariosToAdd">
-              @for (scenario of scenariosToAdd; track scenario) {
+              @for (scenario of scenariosToAdd; track scenario; let index = $index) {
                 <tr class="cursor">
-                  <td>
-                    <button type="button" class="btn btn-outline-primary btn-sm"
-                      (click)="removeScenario(scenario)">
-                      <span class="fa fa-arrow-circle-right"></span>
-                    </button>
-                  </td>
-                  <td class="text-nowrap">{{scenario?.id}}</td>
-                  <td class="scenario-title-cell">{{scenario?.title}}</td>
+                  <td class="text-nowrap">{{scenario?.scenarioId.id}}</td>
+                  <td class="scenario-title-cell">{{scenario?.scenarioId.title}}</td>
                   <td class="p-0">
                     @if (jiraMap.has(scenario.id)) {
                       <span
-                        [ngClass]="getJiraLastExecutionStatusClass(scenario.id)" class="badge">
-                        {{getJiraLastExecutionStatus(scenario.id)}}
+                        [ngClass]="getJiraLastExecutionStatusClass(scenario.scenarioId.id)" class="badge">
+                        {{getJiraLastExecutionStatus(scenario.scenarioId.id)}}
                       </span>
                     }
                   </td>
                   <td class="p-0">
                     @if (jiraMap.has(scenario.id)) {
                       <a  role="button" class="btn btn-sm btn-link me-0"
-                        target="_blank" href="{{getJiraLink(scenario.id)}}" rel="noopener noreferrer"
-                        ngbPopover="{{jiraMap.get(scenario.id)}}" placement="left" (click)="$event.stopPropagation()"
+                        target="_blank" href="{{getJiraLink(scenario.scenarioId.id)}}" rel="noopener noreferrer"
+                        ngbPopover="{{jiraMap.get(scenario.scenarioId.id)}}" placement="left" (click)="$event.stopPropagation()"
                         triggers="mouseenter:mouseleave">
                         <span class="fab fa-jira" aria-hidden="true">
                         </span>
                       </a>
                     }
                   </td>
+                  <td class="pe-5">
```

Comment:
If dataset has a long name (could be up to 64 characters), after select we have :
![image](https://github.com/chutney-testing/chutney/assets/58807088/c120169c-f50e-4820-840c-232ae9f851dc)

Just clip it maybe ?